CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.5.0 / 2020-03-03
+
+* [FEATURE] promauto: Add a factory to allow autoregistration with a local registry. #713
```

Comment:
```suggestion
* [FEATURE] promauto: Add a factory to allow auto registration with a local registry. #713
```
:thinking:

Reply:
In the doc comment, we use `auto-register` once, but the proper way is "automatic registration", which I'll do now.

Reply:
Sorry for being picky ^^ I learned from the best.... (((: